src/plugins/saved_objects/README.md

```diff
@@ -0,0 +1,119 @@
+# Saved Object
+
+The saved object plugin provides all the core services and functionalities of the saved objects, and is utilized by many other plugins such as visualizaton plugin, dashboard plugin and wizard plugin etc.
+
+## Save Relationships to Index Pattern
```

Comment:
Ideally we'd have a general section for relationships and how they're defined, with this as a subsection. But it's fine for now.